third-party/gasnet/Makefile

```diff
@@ -105,10 +105,12 @@ gasnet-config: FORCE
 	$(XTRA_CONFIG_COMMAND)
 	cd $(GASNET_BUILD_DIR) && $(CHPL_GASNET_ENV_VARS) $(GASNET_SUBDIR)/$(CHPL_GASNET_CFG_SCRIPT) --prefix=$(GASNET_INSTALL_DIR) $(CHPL_GASNET_CFG_OPTIONS) --disable-seq --enable-par --disable-parsync $(CHPL_GASNET_CFG_OPTIONS)
 
-gasnet-build: FORCE
-	cd $(GASNET_BUILD_DIR) && $(MAKE) all
+# Temporarily force serial sub-make here, to avoid a file manipulation
+# race that empties gasnet_config.h.
```

Comment:
In that case, it might be worth flagging this more explicitly as an issue in the GASNet Makefiles that we're working around.  The current wording is vague as to whether there's something "wrong" and/or where the blame lies (not that I'm trying to point fingers, just to make it clear that we're working around something and what it is).  GASNet is pretty good about tracking issues, so if they've assigned it an issue number already, maybe we could embed that in the comment so that future devs who come across this (if we forget to pull it) can feel more or less confident that they can retire the -j1 (?).